Functions/InputOutput/xASL_io_dcm2nii.m

```diff
@@ -351,11 +351,10 @@
                 end
 
                 if length(vectorKeep)>1 % add iVolume suffix (if there are multiple)
-                    try
+                    if isempty(niiInstanceNumber)
+						DestFileName = [DestFileName '_' int2str(iVolume)];

```

Comment:
Why not num2str or xASL_num2str, are these not more flexible?
You can choose, and resolve.

Reply:
Fixed.